### PR TITLE
chore(release): 0.2.2 - the start

### PR DESCRIPTION
**Pull request by CallumGrindle**
This release is the first release of the final UniversalProfile setup. It should be seen as a draft and not production ready release! Please use with caution!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
 
+### [0.2.2](https://github.com/lukso-network/universalprofile-smart-contracts/compare/v0.2.1...v0.2.2) (2021-11-01)
+
+
+### Bug Fixes
+
+* android release ([#45](https://github.com/lukso-network/universalprofile-smart-contracts/issues/45)) ([5234ef2](https://github.com/lukso-network/universalprofile-smart-contracts/commit/5234ef2485da4a0d271efc14e108e92c857d5500))
+* release CI fix ([#43](https://github.com/lukso-network/universalprofile-smart-contracts/issues/43)) ([065a5b0](https://github.com/lukso-network/universalprofile-smart-contracts/commit/065a5b08fe68db7142f23874af4ab681842ea6fd))
+* unresolved conflict in readme ([#44](https://github.com/lukso-network/universalprofile-smart-contracts/issues/44)) ([ea057f9](https://github.com/lukso-network/universalprofile-smart-contracts/commit/ea057f999ee4fdd58a3404d7152be920b609c3d2))
+
```
Comment (Hugoo) [inline CHANGELOG.md:13]: Once this is solved let's rewrite this as there are a few broken links and also non pertinent lines in this changelog

**Review by Hugoo — APPROVED** [review]

**Comment by Hugoo** [comment]
Why are there 293 commits in this PR?

**Comment by CJ42** [comment]
As mentioned by @Hugoo Should also remove the matrix in `build-and-test.yml` to only do node version 16?
Testing for node 12 and 14 is overkill. Also, contracts builds do not depend on node version.

Also this would save CI time ! 🕐

**Comment by CallumGrindle** [comment]
Good point @JeneaVranceanu I have added. @Hugoo as discussed the PR shows 293 commits because we use squash merge for PRs so these appear as new commits

File: CHANGELOG.md (hunk 1 above)
Comment (CallumGrindle) [reply CHANGELOG.md:13]: Indeed we can edit this and delete unnecessary tags so we have a clean history

**Pull request merged**